docs: README trim and getting started guide (#104)
ably/ably-chat-kotlin: CONTRIBUTING.md

        - Incomplete Javadoc
8. Add a tag and push to origin - e.g.: `git tag v1.2.4 && git push origin v1.2.4`
9. Create the release on Github including populating the release notes

> 10. Merge any [docs](https://github.com/ably/docs) PRs related to your changes.

Reviewer: ```suggestion
10. Update [kotlin web docs](https://github.com/ably/docs) related to your changes, minimum required change is updating latest version of the SDK under setup section.
```

> 10. Merge any [website docs](https://github.com/ably/docs) PRs related to your changes. At minimum, you must update the setup/installation section to point to the latest SDK version.

Reviewer: ```suggestion
10. Create and merge PR to update [website docs](https://github.com/ably/docs) related to your changes. At minimum, you must update the setup/installation section to point to the latest SDK version.
```

Reviewer: `Merge any` sounds like you have to merge existing PR, but we need to create explicit PR for related changes and then merge it.

Author: I would suggest keeping the original version - we don't want to suggest that you're thinking about writing docs just as you're cutting the release!

Reviewer: Then step should be done before cutting the release, wdyt

Reviewer: Because, it's mandatory to create a PR to update docs ( minimum version update PR is needed )

Author: 4984a07

Reviewer: LGTM
